corehq/toggles.py

```diff
@@ -1919,5 +1919,9 @@ def _commtrackify(domain_name, toggle_is_enabled):
     'restrict_login_as',
     'COVID: Limit allowed users for login as',
     TAG_CUSTOM,
-    namespaces=[NAMESPACE_DOMAIN]
+    namespaces=[NAMESPACE_DOMAIN],
+    description="""
+    Adds a permission that can be set on user roles to only allow those users to "login as" other users that set custom user proper "login_as_user" to the first users' username.
+    For example, if User A has this permission set on their role, the can only login as User B if User B has the custom property "login_as_user" set to "A".
```

Comment:
```suggestion
    For example, if web user a@a.com has this permission set on their role, they can only login as mobile users who have the custom property "login_as_user" set to "a@a.com".
```

Reply:
Previous wording made it sound like this feature prevents web users from logging in as specific users, rather than limiting web users to logging in as a whitelist of mobile users.